.github/workflows/test.yml

```diff
@@ -58,17 +54,12 @@ jobs:
 
       - name: Install pymavlink
         run: |
-          python setup.py build install
+          pip install .
```

Comment:
Isn't this supposed to always be `pip3`?

Reply:
I meant to write `pip`. The distinction can be important when you have a side-by-side installation of `python2` and `python3` to use `pip` or `pip3`, but frankly part of the reason for this PR is to stop *thinking* about python2 vs python3.

`pip install` is the recommendation as per every package page on PyPI (e.g. see below the package title on https://pypi.org/project/pytest/ ).